tests/Makefile

```diff
@@ -1,4 +1,5 @@
 TEST_CFLAGS  += -include $(srcdir)/tests/unittest.h -Wno-sign-compare
+TEST_CFLAGS  += -pg -g
```

Comment:
The only downside I can think of is it'll generate gmon.out when not traced by uftrace.  Maybe no one cares...

Reply:
I also think so.